Plugin.php

```diff
@@ -130,31 +131,35 @@ public function registerSettings()
     {
         return [
             'settings' => [
-                'label'       => 'rainlab.user::lang.settings.menu_label',
-                'description' => 'rainlab.user::lang.settings.menu_description',
+                'label'       => 'winter.user::lang.settings.menu_label',
+                'description' => 'winter.user::lang.settings.menu_description',
                 'category'    => SettingsManager::CATEGORY_USERS,
                 'icon'        => 'icon-cog',
-                'class'       => 'RainLab\User\Models\Settings',
+                'class'       => 'Winter\User\Models\Settings',
                 'order'       => 500,
-                'permissions' => ['rainlab.users.access_settings']
+                'permissions' => ['winter.users.access_settings']
             ]
         ];
     }
 
     public function registerMailTemplates()
     {
         return [
-            'rainlab.user::mail.activate',
-            'rainlab.user::mail.welcome',
-            'rainlab.user::mail.restore',
-            'rainlab.user::mail.new_user',
-            'rainlab.user::mail.reactivate',
-            'rainlab.user::mail.invite',
+            'winter.user::mail.activate',
+            'winter.user::mail.welcome',
+            'winter.user::mail.restore',
+            'winter.user::mail.new_user',
+            'winter.user::mail.reactivate',
+            'winter.user::mail.invite',
         ];
     }
 
     public function registerNotificationRules()
     {
+        if (!class_exists(\Winter\Notify\Classes\Notifier::class)) {
```

Comment:
Isn't this already imported?

Reply:
@LukeTowers it's defined as a `use` case, but it may not exist if the Notify plugin isn't installed, since it's not a "requirement" so to speak.

Reply:
I could have sworn that class_exists will still use the use statements to resolve even if it doesn't actually exist

Reply:
You might be right, but a line like this was added to the `bindNotificationEvents` method, so I assume at one point it was a problem.